proto/Networking/Requests/Social/GetFriendCodeMessage.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 package POGOProtos.Networking.Requests.Social;
 
-message ListFriendStatusMessage {
```

Comment:
???

Reply:
lol moi aussi se truc est sorti dans la 127.1 je crois faudra que je regarde si existe encore dans le dump ... mais ton code na pas etait toujours actualisé avec le mien moi je utilise la plus par du temps les noms utilisé dans le dump

Reply:
Les conventions de nommage dans les api open source ne sont pas les memes que dans l'app, nous avons des conventions plus prédictibles

Reply:
Ok, mais dans mon code toute les séquences friends sont a l'image dump quand j'ai mise a jour ce truc.

Reply:
oui mais ce que je te dis c'est qu'on n'utilise pas les noms de ton dump

Reply:
lol je ne peut pas deviner se que tu aller mettre même si je utiliserais un autre nom les chances possibles aller être faibles

Reply:
C'est les meme depuis le début (avant moi).  
Si le requesttype est SOME_THING alors request = SomeThingMessage et response = SomeThingResponse.

Reply:
est socialactions ???  est manquant  dans requestype je crois https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Enums/SocialAction.proto#L15

Reply:
is this https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Networking/Responses/Social/GetFriendDetailsResponse.proto

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/6c7ac65e79e4bdec9a6ee0a460b4c4be595c73b0#diff-6a394566527964ada874f78db75504a2R4